pkg/scanners/scannerv4/convert.go

```diff
@@ -119,8 +119,6 @@ func setCvss(vuln *storage.EmbeddedVulnerability, cvss *v4.VulnerabilityReport_V
 	errList := errorhelpers.NewErrorList("failed to parse vector")
 	if v2 := cvss.GetV2(); v2 != nil {
 		if c, err := cvssv2.ParseCVSSV2(v2.GetVector()); err == nil {
-			c.ExploitabilityScore = 0.0 // TODO cvssv2.ExploitabilityScore(v2.GetVector())
-			c.ImpactScore = 0.0         // TODO cvssv2.ImpactScore(v2.GetVector())
 			c.Score = v2.GetBaseScore()
```

Comment:
do we still need this call? (similarly, do we want to verify our results match?)

Reply:
Some security sources will provide the score (e.g., RHEL), while others won't. Let's use when provided; otherwise, we the calculate default.

Reply:
(BTW, I don't want to calculate any score in Scanner.)